control/poseidon.txt

```diff
@@ -15,11 +15,11 @@ queryserver_port=24390
 # to the poseidon operate properly !
 # The available server types for now are : Default, bRO_.* (check servertypes.txt)
 # You should modify this if you're having problems with char list.
-server_type=bRO_2016-11-08a
+server_type=vRO_2021-07-13

```

Comment:
need to clear these defaults

Reply:
yep, after vRO and twRO merge i will update the servertypes.txt and also the poseidon.txt

Reply:
[https://photos.google.com/search/_tra_/photo/AF1QipOd33L9wz2uJqY-xB_K2yNm9ugPXyztG3Kp_fex](url)
Please help me vRo